### DISCUSSION TITLE
Feature request: Locally hosting from a thumbdrive or local files?

### DISCUSSION THREAD

**Discussion by edmundronald** [jupyterlite/jupyterlite#606] [Ideas]
It would be nice to run this locally eg. on ipad or laptop, without net connection or server. 
Even nicer if one could edit notebooks and have them persist ...

Edmund

**Comment by psychemedia** [comment]
In passing , there's a PR that seems to offer offline support and ability to install JupyterLite as a webapp, but it's been in the PR queue for a long time so may well have rotted by now?

https://github.com/jupyterlite/jupyterlite/pull/137#issuecomment-854767060
